localization.tsv

```diff
@@ -0,0 +1 @@
+Term	Notes	English
```

Comment:
Not sure if including a localization file is a good choice
- People going through the tutorial could be confused by it, as the tutorial tells them to download a completely different file
- I assume most people would use a tool like Google Sheets to edit their localization files instead of a code/text editor
- Even if the above assumption is incorrect, it's easy enough to create manually thanks to the localization guide

Reply:
I'm old school. I edit the file in visual studio directly.